### PR TITLE
Build for arm64

### PR DESCRIPTION
**Pull request by michbeck100**
Replacement for #397

### CODE REVIEW

File: scripts/build_push_docker.sh [hunk 1]
```diff
@@ -11,12 +11,11 @@ else
...
-
 echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
-docker push ${IMAGE_NAME}:${TAG}
-
-if [ -z "$(echo $TAG | grep -E "beta|master")" ]; then
-  docker tag ${IMAGE_NAME}:${TAG} ${IMAGE_NAME}:latest
-  docker push ${IMAGE_NAME}:latest
+tags=()
+tags+=${IMAGE_NAME}:${TAG}
+if [[ $TAG != *"beta"* ]] && [[ $TAG != *"master"* ]]; then
```
Comment (AgnesToulet) [inline scripts/build_push_docker.sh:17]:
```suggestion
if [ -z "$(echo $TAG | grep -E "beta|master")" ]; then
```

**Review by AgnesToulet — COMMENTED** [review]
Thanks for contributing! I left one small suggestion but apart from that, it looks good.

**Review by AgnesToulet — APPROVED** [review]

**Comment by michbeck100** [comment]
Unfortunately the build failed. I will create another PR to fix this

**Comment by AgnesToulet** [comment]
This should fix it: https://github.com/grafana/grafana-image-renderer/pull/469

**Pull request merged**